generators/client/needle-api/needle-client-angular.js

```diff
@@ -244,4 +244,13 @@ module.exports = class extends needleClientBase {
             this.generator.debug('Error:', e);
         }
     }
+
+    addAdminRoute(route, modulePath, moduleName) {
+        const errorMessage = `${chalk.yellow('Reference to ') + route} ${chalk.yellow('not added to admin routing module.\n')}`;
+        const adminModulePath = `${CLIENT_MAIN_SRC_DIR}app/admin/admin-routing.module.ts`;
+        const adminModuleRoutingEntry = `,\n      {\n        path: '${route}',\n        loadChildren: () => import('${modulePath}').then(m => m.${moduleName})\n      }`;
```

Comment:
You should add line breaks here, it will be easier to read.

And since 
https://github.com/jhipster/generator-jhipster/blob/f54f7a39c4e2a10b92dbe0360023991acc043c25/generators/client/needle-api/needle-client-angular.js#L226-L239
is almost identical I suggest to create the function:
_addRouteNeedle(route, modulePath, moduleName, needleName, filePath, prefix=',')